README.md

```diff
@@ -52,7 +52,7 @@ a pull request to update the list above.**
 
 In addition to PAPR, many of the projects above are also
 hooked up to
-[our instance of](https://homu-projectatomic-ci-infra.e8ca.engint.openshiftapps.com/)
+[our instance of](https://homu-projectatomic-ci.apps.ci.centos.org/)
```

Comment:
This is not quite yet the case. Only rpm-ostree is running there. I think I'll transfer them all over before the week is over, so I'm OK with merging this as is!